addons/map/config.cpp

```diff
@@ -89,6 +89,11 @@ class RscDisplayMainMap {
             onDraw = QUOTE([ctrlParent (_this select 0)] call DFUNC(onDrawMap));
             #include "MapTweaks.hpp"
         };
+        class InitScript: RscText {
+            onLoad = QUOTE(ctrlParent (_this select 0) call (uiNamespace getVariable 'FUNC(initMainMap)'));
```

Comment:
Use `Extended_DisplayLoad_EventHandlers`, AFAIK `RscDisplayMainMap` uses `BIS_fnc_initDisplay`

Reply:
True. It's called `RscDiary` there however, so I should filter all idd != 12 (`IDD_MAIN_MAP`) though.